conduit/pipeline/pipeline.go

```diff
@@ -329,6 +330,18 @@ func (p *pipelineImpl) Init() error {
 		go p.startMetricsServer()
 	}
 
+	signals := make(chan os.Signal, 1)
+	signal.Notify(signals, os.Interrupt)
+
+	go func() {
+		for sig := range signals {
+			if sig == os.Interrupt {
+				p.logger.Infof("Pipeline received interrupt signal, stopping pipeline. p.pipelineMetadata.NextRound: %d", p.pipelineMetadata.NextRound)
+				p.Stop()
+			}
+		}
+	}()
+
```

Comment:
these are the other signals we support in indexer. 
https://github.com/algorand/indexer/blob/cb2f42818f829df3f1f996ca64e3c0a7fd23a255/cmd/validator/core/command.go#L181

Reply:
that's better thanks!

Reply:
bcaa20cd9c456ae03c2441b4cc00de53abda0288

Reply:
But this most recent commit fails tests. First hunch is that this is a real failure.

Reply:
d0a68c697ab82f9ce30b00433b19a75acda1eefb